Loadouts/csat_ctar_pacmech_apex.hpp

```diff
@@ -0,0 +1,298 @@
+#define MSV_EXP "DemoCharge_Remote_Mag:2"
+
+class Car {
+  TransportWeapons[] = {AT,AT_MAG};
+  TransportMagazines[] = {RIFLE_MAG,RIFLE_MAG,CARBINE_MAG,AR_MAG,AR_MAG,GLRIFLE_MAG_HE,BASE_MINE,BASE_EXP};
+  TransportItems[] = {BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL};
+};
+class Tank {
+  TransportWeapons[] = {AT,AT_MAG};
+  TransportMagazines[] = {RIFLE_MAG,RIFLE_MAG,CARBINE_MAG,AR_MAG,AR_MAG,GLRIFLE_MAG_HE,BASE_MINE,BASE_EXP};
+  TransportItems[] = {BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL};
+};
+class Helicopter {
+  TransportWeapons[] = {AT,AT_MAG};
+  TransportMagazines[] = {RIFLE_MAG,RIFLE_MAG,CARBINE_MAG,AR_MAG,AR_MAG,GLRIFLE_MAG_HE,BASE_MINE,BASE_EXP};
+  TransportItems[] = {BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL};
+};
+class Plane {};
+class Ship_F {};
+
+class potato_msv_rifleman { // rifleman
+  uniform[] = {"U_O_T_Soldier_F"};
+  vest[] = {"V_HarnessO_ghex_F"};
+  headgear[] = {"H_HelmetLeaderO_ghex_F"};
+  backpack[] = {"B_FieldPack_ghex_F"};
+  backpackItems[] = {BASE_MEDICAL};
+  weapons[] = {RIFLE};
+  magazines[] = {RIFLE_MAG,BASE_GRENADES};
+  items[] = {BASE_TOOLS};
+  linkedItems[] = {BASE_LINKED};
+  attachments[] = {RIFLE_ATTACHMENTS};
+  opticChoices[] = {"optic_ACO_grn","optic_Arco_blk_F","optic_Arco_ghex_F","ACE_optic_Arco_PIP","optic_Holosight_blk_F"};
+};
+class Fic_Soldier_Carbine: potato_msv_rifleman {// carbine-man
+  weapons[] = {CARBINE};
+  magazines[] = {CARBINE_MAG,BASE_GRENADES};
+};
+class potato_msv_sr: potato_msv_rifleman {// FTL
+  vest[] = {"V_HarnessOGL_ghex_F"};
+  weapons[] = {GLRIFLE};
+  magazines[] = {GLRIFLE_MAG,GLRIFLE_MAG_HE,GLRIFLE_MAG_SMOKE,LEADER_GRENADES};
+  items[] += {LEADER_TOOLS};
+  linkedItems[] += {LEADER_LINKED,BINOS};
+};
+class potato_msv_sl: potato_msv_sr { // SL
+  headgear[] = {"H_HelmetCrew_O_ghex_F"};
+  handguns[] = {PISTOL};
+  magazines[] += {PISTOL_MAG};
+  linkedItems[] = {BASE_LINKED,LEADER_LINKED,RANGE_FINDER};
+  items[] += {RADIO_MR,"H_HelmetLeaderO_ghex_F"};
+};
+class potato_msv_plt: potato_msv_sl { // PLT
+  backpack[] = {"B_Carryall_ghex_F"};
+  headgear[] = {"H_Beret_blk"};
+  items[] += {RADIO_LR};
+};
+class potato_msv_aplt: potato_msv_plt {}; // PLT Assistant
+class potato_msv_coy: potato_msv_plt { // COY
+  headgear[] = {"H_HelmetCrew_O_ghex_F"};
+};
+class potato_msv_acoy: potato_msv_plt{}; // COY Assistant
+class potato_msv_pol: potato_msv_rifleman { // Political Officer
+  vest[] = {"V_BandollierB_ghex_F"};
+  headgear[] = {"H_Beret_blk"};
+  backpack[] = {}; // This loadout looks stupid with a backpack
+  weapons[] = {"hgun_Pistol_heavy_02_Yorris_F"};
+  magazines[] = {"6Rnd_45ACP_Cylinder:3"};
+  items[] += {RADIO_MR,RADIO_MR,BASE_MEDICAL}; // 2 radios
+};
+class potato_msv_AR: potato_msv_rifleman {// AR
+  weapons[] = {AR};
+  magazines[] = {AR_MAG,PISTOL_MAG,BASE_GRENADES};
+  handguns[] = {PISTOL};
+  attachments[] = {};
+  opticChoices[] = {};
+};
+class potato_msv_g: potato_msv_rifleman {// Grenadier
```

Comment:
Probably, but I don't see any reason why the grenadiers/LAT should have different rifles in this loadout. I left the fic for the carbine man there just in case a mission maker wanted to use it.